src/validateRequest.js

```diff
@@ -0,0 +1,33 @@
+const { SUPPORTED_CASES } = require('./constants');
+
+function validateRequest(textToConvert, targetCase) {
+  const errors = [];
+
+  if (!textToConvert) {
+    errors.push({
+      message:
+        'Text to convert is required.'
+        + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!targetCase) {
+    errors.push({
+      message:
+        '"toCase" query param is required.'
+        + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  } else if (!SUPPORTED_CASES.includes(targetCase)) {
+    errors.push({
+      message:
+        'This case is not supported.'
+        + ' Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
```

Comment:
Better to use `join` and your constant with available cases. What if you will support 1 more case. You will need to find and make changes also in this message